### PR TITLE
Fix L1 fat event filters (90x)

### PR DESCRIPTION
**Pull request by rekovic**
90x version of 81x PR  #16782

### CODE REVIEW

**Comment by rekovic** [comment]
please test

**Comment by rekovic** [comment]
please test

**Comment by rekovic** [comment]
+1

**Comment by fwyzard** [comment]
+1

File: EventFilter/L1TRawToDigi/plugins/L1TValidationEventFilter.cc [hunk 1]
```diff
@@ -102,12 +104,13 @@ L1TValidationEventFilter::filter(edm::Event& iEvent, const edm::EventSetup& iSet
...
-  const FEDRawData& l1tRcd = feds->FEDData(1024);
+  const FEDRawData& tcdsRcd = feds->FEDData(1024);
+  const unsigned char *data = tcdsRcd.data();
 
-  const unsigned char *data = l1tRcd.data();
   FEDHeader header(data);
+  evf::evtn::TCDSRecord record((unsigned char *) data);
 
-  bool fatEvent = (header.lvl1ID() % period_ == 0 );
+  bool fatEvent = (record.getHeader().getData().header.triggerCount % period_ == 0 );
```
Comment (davidlange6) [inline EventFilter/L1TRawToDigi/plugins/L1TValidationEventFilter.cc]: The unpacking of fed1024 should go into EventFilter/Utilities/plugins/TcdsRawToDigi.cc which can make a data product for you to consume.
@rekovic

Comment (davidlange6) [inline EventFilter/L1TRawToDigi/plugins/L1TValidationEventFilter.cc]: And as an added bonus, the duplicated code below can be removed too

**Comment by rekovic** [comment]
ooops....will correct.

**Comment by rekovic** [comment]
will rebase

**Pull request closed without merging**